setup.py

```diff
@@ -49,7 +49,9 @@ def _package_doc(name):
           'setuptools',
           'Products.PluginRegistry',
           'Products.GenericSetup',
-          'Zope2 >= 2.12',
+          'Products.StandardCacheManagers',
+          'ZServer',
```

Comment:
Why pull in `ZServer` as an unconditional dependency?  If it is for WebDAV stuff, then we should add a `webdav` extra and ensure that imports are conditional, tests are skipped, etc. if it is missing.